src/gui/sdlmain.cpp

```diff
@@ -4107,21 +4119,25 @@ void Config_Add_SDL() {
 	Pbool = sdl_sec->Add_bool("waitonerror", always, true);
 	Pbool->Set_help("Wait before closing the console if dosbox has an error.");
 
-	Pmulti = sdl_sec->Add_multi("priority", always, ",");
-	Pmulti->SetValue("auto,auto");
+	Pmulti = sdl_sec->Add_multi("priority", always, " ");
+	Pmulti->SetValue("auto auto");
 	Pmulti->Set_help(
-	        "Priority levels for dosbox. Second entry behind the comma is for when dosbox is not focused/minimized.\n"
-	        "pause is only valid for the second entry. auto disables priority levels and uses OS defaults");
-
-	const char *actt[] = {"auto",   "lowest",  "lower", "normal",
-	                      "higher", "highest", "pause", 0};
-	Pstring = Pmulti->GetSection()->Add_string("active", always, "higher");
-	Pstring->Set_values(actt);
-
-	const char *inactt[] = {"auto",   "lowest",  "lower", "normal",
-	                        "higher", "highest", "pause", 0};
-	Pstring = Pmulti->GetSection()->Add_string("inactive", always, "normal");
-	Pstring->Set_values(inactt);
+	        "Priority when in the foreground (first parameter) and background (second parameter).\n"
+	        "   auto:  Let the host operating system manage the priority (valid for both).\n"
+	        "   mute:  Mute the sound when in the background (valid for background setting).\n"
+	        "   pause: Pause operation when in the background (valid for background setting).\n"
+	        "Default setting is: 'auto auto'");
```

Comment:
I'm guessing you're calling foreground state when the window is in focus, and everything else background, right?

Also, I'm a bit confused; aren't the background/foreground (or active/inactive) priority and whether to mute the audio when the window is not in focus orthogonal concepts? They are certainly treated as separate options in WinUAE (see screenshot below).

Although I agree that muting the audio when the window is not in focus _and_ letting the emulation keep running is not overly useful... I guess the name "priority" throws me off a bit too; it's no longer only controlling just the priority, but also the behaviour in the non-active state, so it's a bit confusing perhaps.

![image](https://user-images.githubusercontent.com/698770/173331881-c8247ea4-f244-4ed1-b24c-2b17e206d829.png)